velox/dwio/parquet/writer/Writer.h

```diff
@@ -117,6 +119,9 @@ struct WriterOptions : public dwio::common::WriterOptions {
       "hive.parquet.writer.timestamp_unit";
   static constexpr const char* kParquetHiveConnectorWriteTimestampUnit =
       "hive.parquet.writer.timestamp-unit";
+  /// Parquet datapage version to be used (V1 or V2). Defaults to V2.
```

Comment:
Are you sure it defaults to V2? The Arrow WriterProperties default is V1.

Reply:
@svm1 can you fix this?

Reply:
Corrected, thanks.